### ISSUE TITLE
provide a way to run the tests in node

### ISSUE THREAD

**Issue by k4b7** [utensil-site/available-in-katex#2]
It would be nice if we could run these test in node.  Then whenever a release is done we could take the data and inject it into the Khan/KaTeX README.md.  We could also link back to your site from the README.md.

**Comment by utensil** [comment]
Good idea, sorry I failed to notice this issue until today. I'll look into it.
